types/base/api.d.ts

```diff
@@ -0,0 +1,57 @@
+type Part = {
+    name: string;
+    material: string;
+    color: string;
+}
+
+type InitialsExtra = Record<string, {
+    initials: string;
+    engraving?: string;
+}>
+
+type Spec = {
+    brand: string;
+    model: string;
+    parts: Part[];
+    initials: string;
+    engraving: string;
+    initialsExtra: InitialsExtra;
+    variant?: string;
+    version?: string;
+    description?: string;
+};
+
+type ImportOrderOptions = {
+    brand?: string;
```

Comment:
some of these fields should not be considered optional. Please try to investigate

Reply:
one obvious is the `model`

Reply:
@joamag I believe they're optional because they fallback to the ones that are configured in RIPE SDK.

Reply:
![image](https://user-images.githubusercontent.com/3048875/106266479-5f2a1100-6220-11eb-8975-275fbeab9a96.png)

Reply:
ahh ok, you're right